docs/topics/rn-known-issues.adoc

```diff
@@ -40,4 +40,8 @@ For a complete list of all known issues, see the list of link:https://issues.red
 |link:https://issues.redhat.com/browse/WINDUPRULE-762[WINDUPRULE-762]
 |Web console
 |When an analysis that you run on {ocp-short} from {ProductShortName} fails, logs are not visible.
+
+|link:https://issues.redhat.com/browse/WINDUP-3367[WINDUP-3367]
+|{ocp-short}
+|When you enable FIPs on OpenJDK 11, some pods do not come up when you install {ProductShortName} 5.3.0.Final on {ocp-short}.  
 |====
```

Comment:
@RichardHoch 
FIPS
I would suggest: 
"If you have enabled FIPS mode on OpenJDK 11, some pods do not come up when you install {ProductShortName} 5.3.0.Final on {ocp-short}."

Reply:
@sbeskin-redhat Fixed FIPS; good catach. Other known issues use "when," not "if," in their description. Are you recommending that I change all occurrences of "when" to "if" in the known issues?

Reply:
@RichardHoch 
The issue here is not 'when' per se but twice 'when'. "When you enable FIPS on OpenJDK 11, some pods do not come up" - this 'when' is wrong because nothing happens to the pods when you enable FIPS. The real 'when' is this: "Some pods do not come up when you install..." - that is, the user's action and its problematic result. But the enabled FIPS mode is only a precondition for this problem; it may happen long before the installation attempt. 
You may also say: "When you enable FIPS mode on OpenJDK 11 and then install ..., some pods do not come up."

Reply:
@sbeskin-redhat -- you're right. I went with the last suggestion.